src/root/components/navbar.js

```diff
@@ -208,12 +216,30 @@ function Navbar(props) {
                   autoload={false}
                   noOptionsMessage={getSelectInputNoOptionsMessage}
                   cache={false}
-                  loadOptions={loadOptionsWithDebouncing} />
+                  loadOptions={loadOptionsWithDebouncing}
+                  onKeyDown={(e) => {
+                    if (e.which === 13) {
+                      e.preventDefault();
+                      if (searchTextRef.current?.trim().length > 2) {
+                        setRedirectSearchStrihng(searchTextRef.current);
```

Comment:
```suggestion
                        setRedirectSearchString(searchTextRef.current);
```

Reply:
I have not done any changes to this component